app/views/front_ui/index.html.erb

```diff
@@ -16,18 +16,18 @@
             <h1>MapKnitter</h1>
             <p><i style="color:#aaa;">A Community Atlas hosted by <a href="https://publiclab.org">PublicLab</a></i></p>
             <p>Use a kite, balloon, or pole to take an aerial photo and tell your own visual story of a place. <a href="/about" class="alert-link">Learn more</a></p>
-            <p>
+            <div class="modal-buttons">
             <a class="btn btn-primary btn-lg" href= 'https://store.publiclab.org/collections/mapping-kits'>
                 Get a mapping kit <i class="fa fa-angle-double-right fa-fw" style="font-size:18px;color:white;"></i>
             </a>
             <% if logged_in? %>
-              <a class="btn btn-primary btn-lg" href="<%= new_map_url %>">
+              <a class="btn btn-primary btn-lg" href="<%= new_map_url %>" style="display:flex; align-content:center; flex-wrap:wrap; justify-content:center;">
```

Comment:
You can refactor these duplicated inline styles into `.modal-buttons a` selector.

Reply:
Sure! I'll do that